### PR TITLE
SixKeyBoard refactor

### PR DESCRIPTION
**Pull request by noroadsleft**
- added matrix macro `LAYOUT`
- readme cleanup
- added `info.json`

### CODE REVIEW

File: keyboards/sixkeyboard/keymaps/default/keymap.c [hunk 1]
```diff
@@ -1,12 +1,12 @@
 
-#include "sixkeyboard.h"
+#include QMK_KEYBOARD_H
 #include "matrix.h"
```
Comment (drashna) [inline keyboards/sixkeyboard/keymaps/default/keymap.c]: Not sure we need matrix.h here.

Comment (drashna) [reply keyboards/sixkeyboard/keymaps/default/keymap.c]: Never mind. Checked code, it most likely is.

**Comment by drashna** [comment]
Thanks!

File: keyboards/sixkeyboard/keymaps/default/keymap.c (hunk 1 above)
Comment (drashna) [reply keyboards/sixkeyboard/keymaps/default/keymap.c]: Most of them aren't needed. The keyboard.h includes quantum.h, which includes a majority of the code. 

But in some cases... 
eeprom.h (not eeconfig.h) is one to NOT remove.  userspace h files, and matrix.h are ones to not remove, as well. 

But action_layer.h, etc ... are already included by default.

Comment (noroadsleft) [reply keyboards/sixkeyboard/keymaps/default/keymap.c]: Thanks for the knowledge @drashna. In past, I've occasionally done a local compile with the keyboard's .h file included directly, then another compile using QMK_KEYBOARD_H in place of as many `#include`s as I think I can get away with, then compared the hashes of the .hex files at the end.

**Pull request merged**